app/controllers/maps_controller.rb

```diff
@@ -93,7 +112,7 @@ def access
 
     respond_to do |format|
       format.json do
-        render json: { message: 'Successfully altered edit permissions' }
+        head :ok
```

Comment:
It seems random, and not really necessary to send a string of text. We don't use that string anywhere in the UI.